target/linux/bcm63xx/patches-5.4/805-board_bcm6328-adb-pdg-a4101n-a-000-1a1-ae.patch

```diff
@@ -0,0 +1,70 @@
+Index: linux-5.4.63/arch/mips/bcm63xx/boards/board_bcm963xx.c
+===================================================================
+--- linux-5.4.63.orig/arch/mips/bcm63xx/boards/board_bcm963xx.c
++++ linux-5.4.63/arch/mips/bcm63xx/boards/board_bcm963xx.c
```

Comment:
Please, refresh the patch (make target/linux/refresh).
Also, this should now be integrated into https://github.com/openwrt/openwrt/blob/master/target/linux/bcm63xx/patches-5.4/512-board_bcm6328.patch